Nix/Types.hs

```diff
@@ -48,9 +48,17 @@ atomText :: NAtom -> Text
 atomText (NInt i)  = pack (show i)
 atomText (NBool b) = if b then "true" else "false"
 atomText NNull     = "null"
-atomText (NPath s p)
-  | s = pack ("<" ++ p ++ ">")
-  | otherwise = pack p
+atomText (NPath isFromEnv p)
```

Comment:
This is so a path like `mkPath False "foo"` doesn't get rendered as just `foo`, but instead as `./foo`